<launch>

    <arg name="lidar" default="true" />

    <!-- Start SLAM -->
    <include file="$(find otter_slam)/launch/kingfisher.launch">
        <!--<arg name="lidar" value="$(arg lidar)" />-->
        <arg name="rviz" value="false" />
    </include>

    <!-- Create a map from the cartographer submaps -->
    <node name="cartographer_occupancy_grid_node" pkg="cartographer_ros"
        type="cartographer_occupancy_grid_node" args="-resolution 0.10" />

    <node name="guidance_node" pkg="otter_coverage" type="guidance_node" output="screen">
    </node>

    <!-- Visualize in rviz -->
    <node name="rviz" pkg="rviz" type="rviz" required="true"
        args="-d $(find otter_coverage)/config/coverage.rviz" />

    <!-- Convert Twist messages to Drive messages -->
    <node pkg="vmrc_gazebo" type="twist2thrust.py" name="twist2thrust" output="screen" args="--keyboard">
        <remap from="left_cmd" to="left_thrust_cmd"/>
        <remap from="right_cmd" to="right_thrust_cmd"/>
    </node>

</launch>
